src/bin/keys/CommandRenew.ts

```diff
@@ -51,10 +52,19 @@ class CommandRenew extends CommandPolykey {
         const grpcClient = pkClient.grpcClient;
         const keyMessage = new keysPB.Key();
 
-        const password = await binProcessors.processPassword(
-          passwordPath,
-          this.fs,
-        );
+        let password: string | undefined;
+        try {
+          password = (
+            await this.fs.promises.readFile(passwordPath, 'utf-8')
```

Comment:
This is meant to be a new option in `binOptions`. It's not supposed to just read a file from the password path.

Reply:
We can make it a mandatory option. You have to set it. Alternatively we can also prompt for it.

Reply:
Done with `binOptions.passwordNewFile` and `processNewPassword`.

Reply:
I did it that way because there's a similar parameter for some of the keys commands (e.g. `encrypt` and `decrypt`) that also take a file as a parameter and process it like that. Should this also be made into a bin option?

Reply:
No that's different. Passwords are special cases.